kernel/kernel-api/src/main/java/io/delta/kernel/TransactionBuilder.java

```diff
@@ -69,6 +70,19 @@ TransactionBuilder withTransactionId(
             String applicationId,
             long transactionVersion);
 
+    /**
+     * Set the table properties for the table.
+     *
+     * @param engine     {@link Engine} instance to use.
+     * @param properties The table properties to set. These are key-value pairs that can be used to
```

Comment:
specify the behavior when the table already contains the property with same key. i.e whether it gets replaced or not.

Reply:
Also, we should ideally not update the metadata if the key-value pair matches exactly with what is there in the table.